src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -344,8 +350,18 @@ export class MoreInfoDialog extends LitElement {
     );
     const title = this._childView?.viewTitle || breadcrumb.pop() || entityId;
 
+    const sizeLabel =
+      this.hass?.localize?.(
+        this.large
+          ? "ui.dialogs.more_info_control.reduce_size"
+          : "ui.dialogs.more_info_control.enlarge_size"
+      ) ?? (this.large ? "Restore size" : "Enlarge");
```

Comment:
`localize` will return an empty string when translations are not loaded yet, so the fallback after `??` will never be used.